nginx/minio_proxy_pass.conf

```diff
@@ -1 +1 @@
-proxy_pass http://minio:8080;
+proxy_pass http://minio-server:8080;
```

Comment:
@iamazeem why did you change this? this is the cause for CI error

Reply:
https://github.com/CloudWebManage/cwm-worker-deployment-minio/runs/3333987304?check_suite_focus=true#step:3:2028

Reply:
The nginx pod was crashing due to this.
The `minio-server` was previously used here: https://github.com/CloudWebManage/cwm-worker-deployment-minio/blob/ecf6410e7ba22b8141c58181c0dd3814b2ff16a9/helm/templates/nginx-deployment.yaml#L43-L44

Reply:
yes, for the helm deployment MINIO_PROXY_PASS_HOST is set to minio-server
for docker-compose it's not set and remains with the defailt file

Reply:
Need to revert it then for `docker-compose` if it should remain the same?
So, for helm, `.values.yaml` can be used to configure `MINIO_PROXY_PASS_HOST`, right?

Reply:
No change is needed, I just tested it and helm deployment works. If you had some error it needs to be debugged
the MINIO_PROXY_PASS_HOST should not be changed

Reply:
Reverted. Tested with `docker-compose` and it worked:

Bucket: test
Object: file.txt

/etc/hosts:

```text
# ...
127.0.0.1 example001.com test.example001.com
```

curl:

```shell
$ curl 'http://example001.com:8080/test/file.txt'
axbycz

$ curl 'http://test.example001.com:8080/file.txt'
axbycz
```

nginx logs:

```text
...
nginx_1              | 172.19.0.1 - - [16/Aug/2021:16:17:19 +0000] "GET /test/file.txt HTTP/1.1" 200 7 "-" "curl/7.58.0" "-"
...
nginx_1              | 172.19.0.1 - - [16/Aug/2021:16:17:33 +0000] "GET /file.txt HTTP/1.1" 200 7 "-" "curl/7.58.0" "-"
```

Reply:
The helm deployment is also working. :thinking: 
Tried to reproduce a couple of times but could not.
I might have had something in the environment at that point in time.